conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -228,12 +228,29 @@ func checkRounds(logger *logrus.Logger, catchpointRound, nodeRound, targetRound
 	return false, nil
 }
 
+func (algodImp *algodImporter) needsCatchup(targetRound uint64) bool {
+	if algodImp.mode == followerMode {
+		// If we are in follower mode, check if the round delta is available.
+		_, err := algodImp.getDelta(targetRound)
+		return err != nil
+	} else {
+		// Otherwise just check if the block is available.
+		_, err := algodImp.aclient.Block(targetRound).Do(algodImp.ctx)
+		// If the block is not available, we must catchup.
+		return err != nil
```

Comment:
Maybe log the error before it gets turned into a bool?

Reply:
thanks, updated.